.github/workflows/typescript_tests.yml

```diff
@@ -103,3 +103,70 @@ jobs:
         working-directory: ./sdks/typescript
         env:
           BEAM_SERVICE_OVERRIDES: '{"python:*": "python"}'
+
+  check_gcp_variables:
+    timeout-minutes: 5
+    name: "Check GCP variables"
+    runs-on: ubuntu-latest
+    outputs:
+      gcp-variables-set: ${{ steps.check_gcp_variables.outputs.gcp-variables-set }}
+    steps:
+      - uses: actions/checkout@v3
+      - name: "Check are GCP variables set"
+        run: "./scripts/ci/ci_check_are_gcp_variables_set.sh"
+        id: check_gcp_variables
+        env:
+          GCP_PROJECT_ID: ${{ secrets.GCP_PROJECT_ID }}
```

Comment:
This won't work on prs because secrets aren't available to forks by default. Note that in the Python version this is gated on the secrets being available in https://github.com/apache/beam/blob/b5a0f4895ccf7bd2f8b03217d866e8d0c67f6f52/.github/workflows/build_wheels.yml#L175

Reply:
I've added a guard.